### DISCUSSION TITLE
G29 - PS3 or PS4 mode

### DISCUSSION THREAD

**Discussion by sampie** [berarma/new-lg4ff#67] [Q&A] [answered]
Hi,

When using this driver, does the PS4 mode work or should the wheel be used in PS3 mode?

**Comment by berarma** [comment, marked as answer]
It should always be in PS3 mode to use it on Linux.

**Comment by sampie** [comment]
Is there a particular reason for using PS3-mode in Linux (as Windows uses PS4-mode and dualbooting between the two operating system requires users to remember to change the mode from the wheel every time)?

**Reply by berarma** [reply]
Logitech has released documentation to use the wheel in PS3 mode only. Me and I think others before me couldn't make it work in PS4 mode. I tried to switch the mode from software but it didn't work.

**Reply by sampie** [reply]
I see. I contacted logitech support. They wrote to me that logitech developers can be contacted by feedback functionality in logitech ghub software. I sent a message though ghub asking technical documentation concerning PS4 mode / contact for a logitech developer who could provide needed technical information for PS4 mode in G29. I also gave link for this discussion. Let's see if we can get in touch with a logitech developer or get the needed documentation.

**Reply by mnthnx64** [reply]
@sampie Did you get any response from Logitech? I need to use the wheel in PS4 mode for a project.

**Reply by sampie** [reply]
Unfortuntely no. It might help if other people also would try to contact logitech.
